When installing for the first time, imp.find_module will fail.

Index: image_pipeline-1.12.15/image_view/cfg/ImageView.cfg
===================================================================
--- image_pipeline-1.12.15.orig/image_view/cfg/ImageView.cfg
+++ image_pipeline-1.12.15/image_view/cfg/ImageView.cfg
@@ -2,8 +2,6 @@
 
 PACKAGE='image_view'
 try:
-    import imp
-    imp.find_module(PACKAGE)
     from dynamic_reconfigure.parameter_generator_catkin import *
 except:
     import roslib; roslib.load_manifest(PACKAGE)
